nltk/parse/dependencygraph.py

```diff
@@ -49,14 +60,11 @@ def __init__(self, tree_str=None, cell_extractor=None, zero_based=False, cell_se
         are split by whitespace.
 
         """
-        self.nodes = defaultdict(lambda: {'deps': defaultdict(list)})
+        self.nodes = defaultdict(lambda: deepcopy(DependencyGraph.default_node))
```

Comment:
I would not use deepcopy (it feels like magic to me) here, instead I would just specify all the needed items inside of lambda, so
``` python
{'deps': defaultdict(list)}
```
becomes

```python
{
    'deps': defaultdict(list)},
    'address': None,
    'word': None,
    ...
}
```
alternatively one could define a function that returns a default node and pass it to the default dict.

Ideally, I think, node should be an object, not a dictionary.